### ISSUE TITLE
Multiple pages shown from the very first Prairie tiff file

### ISSUE THREAD

**Issue by wuzhengherbert** [MouseLand/suite2p#116]
Hi, I acquire my raw tiffs with PrairieView. The very first tiff contains all the acquisition information and is larger than all the other tiffs. The problem is that 320 frames will come from the first tiff when I load it into suite2p.  320 is the number of frames I acquire per trial. When I read that tiff into matlab or imageJ, it only contains one page.

I tried to attach the tiff file but github doesn't support tif.

Thank you!
Zheng

**Comment by carsen-stringer** [comment]
Can you upload the tif to google drive or dropbox and share the link please? Sorry we don't use prairieview so I'm not sure what to expect

**Comment by wuzhengherbert** [comment]
Thanks for the help! Here is the dropbox link:

https://www.dropbox.com/s/p5naboeof60fd67/TSeries-20170508-D43-001_Cycle00001_Ch2_000001.ome.tif?dl=0

**Comment by carsen-stringer** [comment]
That is one tricky tiff :) I've made sure that suite2p checks the length of the tiff before it reads it (which is correct, using skimage.external.tifffile.TiffFile), and then cropping the output of skimage.external.tifffile.imread (which is 320 for some reason). 

Btw I've only updated the github version. To run it, download the repository from github and run 'python -m suite2p' from inside the repository.
